### DISCUSSION TITLE
Ability to give a Dashboard a name and description

### DISCUSSION THREAD

**Discussion by JonBetts** [jobrunr/jobrunr#226] [Ideas]
Hi Ronald,

I am attempting to get multiple Jobrunr Dashboards running on the same Tomcat instance and Java Virtual Machine (obviously started up under different ports). Is it at all possible to be able to allow for some custom text to be added to the Dashboard pages so that I can differentiate between the Dashboards? Otherwise I'll have to keep checking which is which based on the ports I've assigned them to, and if anyone else wants to access the Dashboards, I want them to be sure which Dashboard they are looking at.

Many thanks in advance! 🙌

**Comment by rdehuyss** [comment]
Currently this is not possible but this would not be too difficult. You're welcome to open a PR or a feature request for this.

**Reply by JonBetts** [reply]
Thanks Ronald. I'll create a feature request for the time being, and if I get a chance in the near future, I'll have a look at adding a PR 😃
